src/code.h

```diff
@@ -109,6 +109,25 @@ EXPORT_INLINE Obj VALUES_BODY(Obj body)
     return BODY_HEADER(body)->values;
 }
 
+
+/****************************************************************************
+**
+*F  NewStat( <type>, <size> ) . . . . . . . . . . .  allocate a new statement
+**
+**  'NewStat'   allocates a new   statement memory block  of  type <type> and
+**  <size> bytes.  'NewStat' returns the identifier of the new statement.
+**
+**  NewStatWithProf( <type>, <size>, <line>, <file> ) allows the line number
+**  and fileid of the statement to also be specified, else the current line
+**  and file when NewStat was called is used. line=0, file=0 is used
+**  to denote a statement which should not be tracked.
+*/
+Stat NewStatWithProf(UInt type, UInt size, UInt line);
```

Comment:
Better comment:
```
/****************************************************************************
**
*F  NewStatOrExpr(<type>,<size>,<line>,<file>) . . . allocate a new statement
**
**  'NewStatOrExpr' allocates a new statement or expressions memory block of
**  type <type> and with <size> bytes. It also records the line number <line>
**  and file id <file> as source of the statement for profiling. It returns
**  the offset of the new statement.
**
**  Callers may pass zero for both <line> and <file> to denote a statement
**  which should not be tracked by the profiling code.
*/
```
and then rename `NewStatWithProf` to `NewStatOrExpr` globally (could be in a commit of its own, don't care).